cli/server/server.go

```diff
@@ -105,6 +106,13 @@ func NewCommands() []*cli.Command {
 					Action:    dumpDB,
 					Flags:     cfgCountOutFlags,
 				},
+				{
+					Name:      "dump-bin",
```

Comment:
Maybe not `dump`, but `fetch`.

Reply:
@roman-khimov, why `fetch`? This command saves dump of every block to a separate file, it doesn't fetches them from the network.